src/plugins/intel_gna/src/transformations/aszp_decomposition.cpp

```diff
@@ -0,0 +1,231 @@
+#include "transformations/aszp_decomposition.hpp"
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <openvino/cc/ngraph/itt.hpp>
+#include "backend/gna_limitations.hpp"
+#include "aszp_decomposition.hpp"
+#include <memory>
+
+using namespace ngraph;
+using namespace op;
+
+namespace ov {
+namespace intel_gna {
+namespace pass {
+
+static std::tuple<int64_t, int64_t, int64_t, int64_t, int64_t, int64_t> ExtractPadding(ov::CoordinateDiff pads_begin, ov::CoordinateDiff pads_end) {
+	auto Hbegin = pads_begin[0];
+	auto Hend   = pads_end[0];
+	auto Wbegin = pads_begin[1];
+	auto Wend   = pads_end[1];
+
+	return std::make_tuple(
+		Hbegin, Hend, Wbegin, Wend, 
+		Hbegin > Hend ? Hbegin - Hend : Hend - Hbegin, 
+		Wbegin > Wend ? Wbegin - Wend : Wend - Wbegin);
+}
+
+
+
+std::shared_ptr<ngraph::opset8::Transpose> CreateTranspose(const Output<Node>& input) {
+    return std::make_shared<ngraph::opset8::Transpose>(input,
+		ngraph::op::Constant::create(element::Type_t::i64, Shape{2}, {1, 0}));
+}
+
+std::shared_ptr<ngraph::opset8::Reshape> CreateReshape(const Output<Node>& input, uint64_t ndims, ov::Shape shape) {
+	return std::make_shared<ngraph::opset8::Reshape>(input, 
```

Comment:
please change `ngraph::opset8::` to `ov::opset11::` if possible

Reply:
done